packages/udes-ui-validation/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "udes-ui-validation",
+  "version": "0.0.1",
+  "description": "Package to validate react component fields.",
+  "main": "webpack.config.js",
+  "scripts": {
+    "jest:lerna": "jest --coverage",
+    "test": "lerna run jest:lerna",
+    "lint": "eslint src -c .eslintrc.json --ext js,jsx,ts,tsx; exit 0",
+    "check": "lint-staged"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/unipartdigital/udes-ui.git"
+  },
+  "keywords": [
+    "udes",
+    "udes-ui",
+    "udes-ui-validator"
+  ],
+  "author": "Robert Smith <robert.smith@unipart.io>",
+  "license": "GPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/unipartdigital/udes-ui/issues"
+  },
+  "homepage": "https://github.com/unipartdigital/udes-ui#readme",
+  "dependencies": {
+    "react-scripts": "3.2.0",
+    "styled-components": "4.4.1",
+    "validator": "12.1.0"
+  },
+  "devDependencies": {},
+  "jest": {
+    "roots": [
```

Comment:
Jest config seems quite different to the other packages, is there a reason for this?

Reply:
the config from other files didn't work if i remember correctly, but i can take bits of code out to improve it potentially, see what exactly causes it to break

Reply:
copied and pasted from one of the other packages and _seems_ to still work